src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -556,9 +556,14 @@ func (c controller) validateConsolePod() bool {
 }
 
 func (c controller) waitingForClusterVersion() error {
-	isClusterVersionAvailable := func() bool {
+	isClusterVersionAvailable := func(timer *time.Timer) bool {
 		newCvoStatus := c.validateClusterVersion()
 		if c.cvoStatus.isAvailable != newCvoStatus.isAvailable || (c.cvoStatus.message != newCvoStatus.message && newCvoStatus.message != "") {
+			if !timer.Stop() {
+				<-timer.C
+			}
+			timer.Reset(WaitTimeout)
```

Comment:
what change are u looking at?

Reply:
Message/Availability change.
Basically, that timeout indicates static-waiting and not on-going changes.

Reply:
But message can be:
"Multiple Errors"....

Reply:
We can start with this check and in the future make a smarter check. Unless you think it's necessary.

Reply:
@YuviGold do you have an assisted-service log with this running, for reference?